server/src/main/scala/net/wiringbits/tasks/NotificationsTask.scala

```diff
@@ -0,0 +1,35 @@
+package net.wiringbits.tasks
+
+import akka.actor.ActorSystem
+import com.google.inject.Inject
+import net.wiringbits.actions.{GetPendingNotificationsAction, SendNotificationAction}
+import net.wiringbits.config.NotificationsConfig
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.ExecutionContext
+
+class NotificationsTask @Inject() (
+    notificationsConfig: NotificationsConfig,
+    getPendingNotifications: GetPendingNotificationsAction,
+    sendNotificationAction: SendNotificationAction
+)(implicit
+    ec: ExecutionContext,
+    actorSystem: ActorSystem
+) {
+  val logger = LoggerFactory.getLogger(this.getClass)
+
+  logger.info("Starting the notifications task")
+  actorSystem.scheduler.scheduleAtFixedRate(
```

Comment:
Let's use `scheduleOnce` instead, when it finishes we just schedule it once again, this way, we won't have two concurrent tasks running concurrently which will likely break the workflow, [example](https://github.com/X9Developers/block-explorer/blob/develop/server/app/com/xsn/explorer/tasks/PollerSynchronizerTask.scala#L33)

Also, in order to write test, we will have to create a `run` method that returns a handler to stop the task but we can check that later.